projects/plugins/jetpack/json-endpoints/class.wpcom-json-api-get-site-endpoint.php

```diff
@@ -200,6 +200,7 @@ class WPCOM_JSON_API_GET_Site_Endpoint extends WPCOM_JSON_API_Endpoint {
 		'launchpad_checklist_tasks_statuses',
 		'wpcom_production_blog_id',
 		'wpcom_staging_blog_ids',
+		'has_promote_widget',
```

Comment:
Would the `has_promote_widget` be removed from the `/me/` endpoint in a separate PR?